libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -137,6 +137,18 @@ def construct_analyzer_cmd(self, res_handler):
                                          '-analyzer-disable-checker',
                                          '-Xclang', checker_name])
 
+            if config.ctu_dir:
+                analyzer_cmd.extend(['-Xclang', '-analyzer-config',
+                                     '-Xclang', 'xtu-dir=' + config.ctu_dir,
+                                     '-Xclang', '-analyzer-config',
+                                     '-Xclang', 'reanalyze-xtu-visited=true'])
+                if config.ctu_in_memory:
+                    analyzer_cmd.extend(['-Xclang', '-analyzer-config',
+                                         '-Xclang',
+                                         'xtu-reparse=' +
+                                         os.path.abspath(config.log_file[0])])
+            # TODO Make it work on multiple logfiles
```

Comment:
(Note: The analyzer command is constructed for each and every build action separately. Maybe this fact could be used to handle this TODO.)